ace/sources.py

```diff
@@ -714,24 +736,39 @@ def parse_article(self, html, pmid=None, **kwargs):
         if not soup:
             return False
 
-        # Extract tables
+        # Extract table; going to take the approach of opening and parsing the table via links
+        # To download tables, we need the content URL and the number of tables
```

Comment:
A comment like this is fine if its explaining what you're doing, not a TODO comment